memcached.c

```diff
@@ -231,6 +231,76 @@ static rel_time_t realtime(const time_t exptime) {
     }
 }
 
+/*
```

Comment:
this was just moved to make the clock bits available to the protocol?

for future reference it's a bit harder to review when chunks of code are moved and changed at the same time.